src/js/multiDomainFirstParties.js

```diff
@@ -2999,6 +2999,7 @@ let multiDomainFirstPartiesArray = [
     "azurerms.com",
     "azuresynapse.net",
     "bing.com",
+    "platform.bing.com",
```

Comment:
MDFP lists only function at the level of [eTLD+1 domains](https://en.wikipedia.org/wiki/Public_Suffix_List), and `bing.com` is already on the list.

Reply:
Ahh, I see. I guess I was trying to split the difference between this file and yellowlist.txt - missed the mark on both!